docs/src/frontend/index.md

```diff
@@ -1007,6 +1007,24 @@ stdlib.assert(p)
 
 Throws an exception if not given `{!js} true`.
 
+Example:
+
+```js
+load: /examples/argz/index.mjs
+md5: d98c02ebf76e5ce55b6dec475c82539e
+range: 4 - 7
+```
+
+This code checks the arguments passed to the `{!cmd} ./reach run` command.
+Lines 6 and 7 would not throw an exception if `{!cmd} ./reach run index hello "Mr. Postman"` is run in the terminal.
+This is because `{!js} process.argv` would return the following `{!rsh} array`:
+
+```js
+[ '/usr/local/bin/node', '/app/index.mjs', 'hello', 'Mr. Postman' ]
```

Comment:
'Hello' should be capitalized.

Reply:
Looks like you need to rebase. `hs-test` is failing due to `Fix_Switch_DoubleCase.txt`